addon/components/ui-text-input/template.hbs

```diff
@@ -1,4 +1,5 @@
 {{input
+  class=(if (not (is-empty this.width)) "has-width")
```

Comment:
Thinking - is there a is-not-empty helper? Just to make this a bit cleaner?

Reply:
Not sure why it didn't transfer the comments from the other discussion, but there isn't a is-not-empty nor not-empty in ember-truth-helpers - https://github.com/jmurphyau/ember-truth-helpers - unless I am looking in the wrong place? `if this.width` also works and would be cleaner too

Reply:
It didn't transfer the comments from the other discussion because it was discussed about the ts file, not the template